megatron/model/transformer.py

```diff
@@ -588,9 +588,11 @@ def __init__(self, config, layer_number,
             local_attn = FlashSelfAttention(causal=True, attention_dropout=config.attention_dropout)
         else:
             local_attn = CoreAttention(self.layer_number, config, self.attn_mask_type)
-
-        self.enable_ds_sequence_parallel = parallel_state.get_sequence_parallel_world_size() > 1 \
-                                           or args.force_ds_sequence_parallel
+        if hasattr(args, 'ckpt_transfer') and args.ckpt_transfer:
```

Comment:
I did not notice `--ckpt_transfer` in the argument parsing code. How is this attribute added to `args`?

Reply:
> I did not notice `--ckpt_transfer` in the argument parsing code. How is this attribute added to `args`?

 I understand that you are likely busy with many responsibilities, but I would greatly appreciate your feedback on this PR when you get a chance.

Reply:
> > I did not notice `--ckpt_transfer` in the argument parsing code. How is this attribute added to `args`?
> 
> I understand that you are likely busy with many responsibilities, but I would greatly appreciate your feedback on this PR when you get a chance.

Hi, @uygnef , thank you for great your work! I am trying to use this script for convert HF LLAMA to Megatron-Deepspeed format and I met the same error `AssertionError: sequence parallel group is not initialized`. Do you solve this issue?

Reply:
> > I did not notice `--ckpt_transfer` in the argument parsing code. How is this attribute added to `args`?
> 
> I understand that you are likely busy with many responsibilities, but I would greatly appreciate your feedback on this PR when you get a chance.

Hi, @uygnef, I changed `ckpt_transfer` parameter so it works. But it seems the output format is Megatron-LM format not Megatron-DeepSpeed format?

Reply:
Hi @uygnef, thank you so much for this pr! Would it be possible for you to provide an example of a launch script for it?